pet-store-vanilla-js/index.html

```diff
@@ -3,6 +3,140 @@
   <head>
     <meta charset="UTF-8" />
```

Comment:
The add modal should be able to opened even after the view/edit modal has a load error.